wire-ios/Wire-iOS/Sources/AnalyticsTrackingManager/TrackingManager.swift

```diff
@@ -71,14 +101,29 @@ final class TrackingManager: TrackingInterface {
 
     func enableAnalytics() async throws {
         try await sessionManager.makeEnableAnalyticsUseCase()?.invoke()
-        ExtensionSettings.shared.disableAnalyticsSharing = false
         AVSFlowManager.getInstance()?.setEnableMetrics(true)
+        journal?[.isAnalyticsTrackingConsentGiven] = true
```

Comment:
Ok I see now that this flag is concerning whether analytics is enabled. Perhaps we should name is `isAnalyticsTrackingEnabled`